src/frontend/overlay/src/components/organisms/widgets/Scoreboard.tsx

```diff
@@ -190,31 +196,50 @@ export const ScoreboardRow = ({
                         maxScore={contestData?.problems[i]?.maxScore}
                     />
                 ))}
-        </ScoreboardRowWrap>
-    );
-};
-
-type PositionedScoreboardRowProps = {
-    zIndex: number;
-    pos: number;
-};
+            </>
+        );
+    },
+);
+
+export const ScoreboardRow = React.memo(
```

Comment:
Do we need React.memo here? Can we rely on react compiler instead?
If you want to force it to have memo here, you can annotate it with "use memo"
https://react.dev/reference/react-compiler/directives/use-memo